app/index.js

```diff
@@ -172,6 +172,11 @@ app.on('ready', () =>
       // expose to plugins
       app.createWindow = createWindow;
 
+      // check if should be set as default ssh protocol client
+      if (config.getConfig().defaultSSHApp == true && !app.isDefaultProtocolClient('ssh')) {
```

Comment:
you can just do `config.getConfig().defaultSShApp` no need to test if `true` or else if you need to implicitly test it you will need to `config.getConfig().defaultSShApp === true`

Reply:
Yup, my bad, just fixed it